examples/passage_reranker/bert.py

```diff
@@ -0,0 +1,237 @@
+# Copyright 2019 The Forte Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Utils of BERT Modules.
+"""
+import os
+from typing import Optional, cast
+
+import torch
+from torch.nn import Parameter
+
+from texar.torch.modules.pretrained import PretrainedBERTMixin
+from texar.torch.modules.encoders import BERTEncoder
+from texar.torch.modules.classifiers import BERTClassifier
+from texar.torch.hyperparams import HParams
+
+__all__ = [
+    "FineTunedBERTEncoder",
+    "FineTunedBERTClassifier"
+]
+
+
+class FineTunedBERTClassifier(BERTClassifier, PretrainedBERTMixin):
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+        hparams = self.default_hparams()
+        hparams.update(kwargs)
+
+        self.load_pretrained_config(hparams=hparams)
+
+        super().__init__(pretrained_model_name=self.pretrained_model_name,
+                         cache_dir=self.cache_dir)
+
+        self.init_pretrained_weights(prefix=self._hparams.prefix)
+
+    def default_hparams(self):
+        params = super().default_hparams()
+        params.update(self.kwargs)
+        return params
+
+    def load_pretrained_config(self,
```

Comment:
This looks complex even if we are using Texar. Why is this? Can Texar handle most of the model loading?